common-api/src/main/java/org/pmiops/workbench/config/WorkbenchConfig.java

```diff
@@ -73,8 +73,12 @@ public static WorkbenchConfig createEmptyConfig() {
     // The full table name for the BigQuery billing export, which is read from by the free-tier
     // usage tracking cron endpoint.
     public String exportBigQueryTable;
-    // The default limit to apply to free-credit usage in this environment.
-    public Double defaultFreeCreditsLimit;
+    // DEPRECATED.  Renamed to defaultFreeCreditsDollarLimit.
```

Comment:
nit: as a convention would be nice to also include in the comment: "Remove after `https://github.com/all-of-us/workbench/pull/2920` reaches production."

Reply:
sure